docs/platforms/python/feature-flags/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+description: With Feature Flags, Sentry tracks flag evaluations in your application and reports their state on error. Sentry will also record an audit log of feature flag changes and report any suspicious changes that may have triggered an error.
```

Comment:
are there any python platforms that should be included/excluded? there's an allow list here that should match with the docs: https://github.com/getsentry/sentry/pull/81371

Reply:
All python platforms should be included.